src/content/app/tools/blast/BlastPage.scss

```diff
@@ -1,6 +1,7 @@
 @import 'src/styles/common';
 
 .blastPage {
+  --blast-container-max-width: 1800px;
```

Comment:
There are a couple of other places where the magic 1800px value is used. Some of them, on our BLAST pages:

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/6834224/212639553-ad35843d-92b4-4428-9fd3-9547dbd26e0f.png">

While we can't use CSS variables in media queries in `_breakpoints.scss` and `globalConfig.ts`, we should probably update other BLAST SCSS files to use the CSS variable above.